storage-bigtable/src/lib.rs

```diff
@@ -392,12 +400,12 @@ impl LedgerStorage {
     pub async fn new(
         read_only: bool,
         timeout: Option<std::time::Duration>,
-        credential_path: Option<String>,
+        credential_option: Option<CredentialOption>,
```

Comment:
Let's add a new constructor, and leave `LedgerStorage::new()` unbroken. Something like:
```
pub async fn new_with_stringified_credential(credential: String) -> Result<Self> {
    Self::new_with_config(LedgerStorageConfig {
            read_only,
            timeout,
            credential_type: CredentialType::Stringified(credential),
            ..LedgerStorageConfig::default()
        })
        .await
}
```